modules/end-user-guide/partials/proc_mounting-a-secret-as-an-environment-variable-into-a-workspace-container.adoc

```diff
@@ -1,23 +1,61 @@
 // Module included in the following assemblies:
-//
 // mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
 
 [id="mounting-a-secret-as-an-environment-variable-into-a-workspace-container_{context}"]
 = Mounting a secret as an environment variable into a workspace container
 
-The following section describes how to mount a {platforms-name} secret from the user’s {orch-namespace} as an environment variable, or variables, into single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as an environment variable (or variables) in single-workspace or multiple-workspace containers.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}. See {link-installing-an-instance}.
 
 .Procedure
 
-. In the {platforms-namespace} where a {prod-short} workspace will be created, generate a new {platforms-name} secret.
-
-* The labels of the secret that is about to be generated must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. By default, it is a set of two labels: 
-+
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new secret in the {platforms-namespace} where a new workspace is deployed.
+. Define the default behavior for mounting secrets in the `automount-workspace-secret` annotation, which takes a boolean value:
+** `'true'` enables automatic mounting into all workspace containers.
+** `'false'` disables automatic mounting into all workspace containers. (To mount a secret, a user must request it in the devfile component by using the `automountWorkspaceSecrets:true` property.)
+. Set the name of the secret's environment variable in the `env-name` annotation.
```

Comment:
```suggestion
. Set the name of the secret environment variable in the `env-name` annotation.
```

Reply:
@rkratky I understand your point on why we need to drop the apostrophe, but I only added the apostrophe to separate `the secret` and the `environment variable`. Otherwise, it can be misunderstood as an `environment variable` that is `secret` because `secret` is also an adjective (in addition to being a noun as intended above).
As an alternative, we may get something wordier here:
`". Set the name of the environment variable of the secret in the `env-name` annotation."`